lib/utils/__tests__/narrowFixRange.test.mjs

```diff
@@ -306,6 +306,71 @@ describe('narrowFixRange', () => {
 				'a {--foo: abcabc}',
 			);
 		});
+		test('declaration with trailing semicolon', () => {
+			const node = postcss.parse('a { opacity: 45%; }');
+
+			const fixData = {
+				range: [4, 17],
+				text: 'opacity: 0.45;',
+			};
+
+			const narrowRange = narrowFixRange(node, fixData);
+
+			expect(narrowRange).toMatchObject({
+				range: [13, 16],
+				text: '0.45',
+			});
+
+			expect(applyFixData('a { opacity: 45%; }', narrowRange.text, narrowRange.range)).toBe(
+				'a { opacity: 0.45; }',
+			);
+		});
+		test('atrule with trailing semicolon', () => {
```

Comment:
```suggestion
		});

		test('atrule with trailing semicolon', () => {
```

Some whitespace to make the boundaries between tests more clear.

Can you apply this between all `test()` calls?